setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = versations
+version = 0.1.1
+description = A package for syncing, logging, and sending messages to Matrix rooms.
+long_description = file:README.md
+long_description_content_type = text/markdown
+
+[options]
+packages = find:
```

Comment:
You still need to add the following section to `setup.cfg` in order for `find:` to function properly:

```ini
[options.packages.find]
where = src
```

Without this, no code is installed.  To see exactly what parts of a package get installed, you can:

* Install [`build`](https://github.com/pypa/build)
* Run `python3 -m build --wheel`
* Run `zipinfo` on the resulting `dist/*.whl` file

Reply:
Is it correct that the files in src and versations are doubled up like this?

$ zipinfo dist/*.whl
Archive:  dist/versations-0.1.1-py3-none-any.whl
Zip file size: 17010 bytes, number of entries: 16
-rw-r--r--  2.0 unx        0 b- defN 23-Feb-15 14:56 src/__init__.py
-rw-r--r--  2.0 unx     4108 b- defN 23-Mar-21 21:17 src/client.py
-rw-r--r--  2.0 unx     8816 b- defN 23-Feb-15 14:56 src/client_callbacks.py
-rw-r--r--  2.0 unx     5694 b- defN 23-Mar-21 21:17 src/main.py
-rw-r--r--  2.0 unx     2075 b- defN 23-Mar-17 17:37 src/session.py
-rw-r--r--  2.0 unx        0 b- defN 23-Feb-15 14:56 versations/__init__.py
-rw-r--r--  2.0 unx     4108 b- defN 23-Mar-21 21:17 versations/client.py
-rw-r--r--  2.0 unx     8601 b- defN 23-Mar-23 14:28 versations/client_callbacks.py
-rw-r--r--  2.0 unx     5696 b- defN 23-Mar-23 14:27 versations/main.py
-rw-r--r--  2.0 unx     2075 b- defN 23-Mar-17 17:37 versations/session.py
-rw-r--r--  2.0 unx     1073 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/LICENSE
-rw-r--r--  2.0 unx     1047 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/METADATA
-rw-r--r--  2.0 unx       92 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/WHEEL
-rw-r--r--  2.0 unx       51 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/entry_points.txt
-rw-r--r--  2.0 unx       11 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/top_level.txt
-rw-rw-r--  2.0 unx     1250 b- defN 23-Mar-24 12:21 versations-0.1.1.dist-info/RECORD

Reply:
No.  I suggest doing `git clean -dXf` to remove the `build/` cache and running `python3 -m build` again.

Reply:
that worked